monkey/monkey_island/cc/resources/telemetry.py

```diff
@@ -60,8 +61,9 @@ def post(self):
 
         process_telemetry(telemetry_json)
 
-        telem_id = mongo.db.telemetry.insert(telemetry_json)
-        return mongo.db.telemetry.find_one_or_404({"_id": telem_id})
+        TelemetryModel.save_telemetry(telemetry_json)
+
+        return {}, 201
```

Comment:
I don't know and couldn't figure out why the telemetry was sent back after insert. Since the codebase were not catching any errors before, I don't see why it's relevant to know which error the failed save method gives out, because in any case, flask spits out 500 error.